script.js

```diff
@@ -9,3 +9,31 @@
 // Add a `facial-hair` property to `instructor`.
 
 // Log the value of the `facial-hair` property to the console.
+
+$(document).ready(function(){
+  console.log("hello");
+
+var colors = ['red','yellow', 'green'];
+  console.log(colors[2]);
+
+  $(".bulb").on("click", function(evt){
+    evt.preventDefault();
+    console.log("click me");
+    var colors = $(".bulb").colors[];
+  })
+//
+//       for(i=0, i<0, i++){
+//           console.log("hello");
+//           colors[i]= ("my favorite color is" + [i]);
+//       }
+//   }
```

Comment:
for (var i = 0; i < colors.length, i++){
  console.log("my favorite color is " + colors[i])
}